### PR TITLE
Added horizontal FOV and videostream light spectrum enum

### PR DESCRIPTION
**Pull request by markusachtelik**
[outdated:]
This PR adds the interfaces to subscribe to VIDEO_STREAM_STATUS messages from the camera. https://mavlink.io/en/messages/common.html#VIDEO_STREAM_STATUS 

[new]
after discussions, we decided to not duplicate the redundant mavlink messages VIDEO_STREAM_STATUS and VIDEO_STREAM_INFORMATION. Instead, we extend the current structure, and update it whenever data from either of these messages arrives. 
PR in mavsdk will follow. 
@julianoes , @JonasVautherin  can you take a look?

### CODE REVIEW

File: protos/camera/camera.proto [hunk 1]
```diff
@@ -65,6 +65,10 @@ service CameraService {
      * Subscribe to video stream info updates.
      */
     rpc SubscribeVideoStreamInfo(SubscribeVideoStreamInfoRequest) returns(stream VideoStreamInfoResponse) {}
+    /*
+     * Subscribe to video stream info updates.
```
Comment (julianoes) [inline protos/camera/camera.proto:69]:
```suggestion
     * Subscribe to video stream status updates.
```

File: protos/camera/camera.proto [hunk 2]
```diff
@@ -311,6 +320,18 @@ message VideoStreamInfo {
     Status status = 2; // Current status of video streaming
 }
 
+// Type for video stream status. Difference to VideoStreamSettings: Status gets sent whenever there is a change, e.g. fov of the camera. Settings get queried usually once during startup.
+message VideoStreamStatus{
+    bool is_running = 1; // Video stream is running / not running
```
Comment (julianoes) [inline protos/camera/camera.proto:325]: This is e.g. a duplicate with `Status` (`STATUS_NOT_RUNNING` and `STATUS_IN_PROGRESS`).

File: protos/camera/camera.proto [hunk 3]
```diff
@@ -311,6 +320,18 @@ message VideoStreamInfo {
...
+    bool is_thermal = 2; // Indicates whether the stream is from thermal imaging (true) or visible light (false)
+    float frame_rate_hz = 3; // Frames per second
+    uint32 horizontal_resolution_pix = 4; // Horizontal resolution (in pixels)
+    uint32 vertical_resolution_pix = 5; // Vertical resolution (in pixels)
+    uint32 bit_rate_b_s = 6; // Bit rate (in bits per second)
+    uint32 rotation_deg = 7; // Video image rotation (clockwise, 0-359 degrees)
+    uint32 horizontal_fov_deg = 8; // Horizontal field of view in degrees
```
Comment (julianoes) [inline protos/camera/camera.proto:332]: These two types we should change to `float` anyway, either here or in `VideoStreamSettings` or both.

**Review by julianoes — CHANGES_REQUESTED** [review]
We now basically have the same mess as in the mavlink messages. We have the same fields twice, once as status and one as information/settings. I don't think that's a great.

If you look at `VideoStreamInfo` it already contains dynamic fields such as `Status` (running/not running), so I would probably suggest to extent that rather than duplicate it.

For the MAVSDK API we don't necessarily need to distinguish between the static and dynamic data because we don't care as much about the overhead, or the amount of bytes needed to be transmitted, so I would say one big structure will all information is sane.

File: protos/camera/camera.proto [hunk 4]
```diff
@@ -296,6 +296,7 @@ message VideoStreamSettings {
     uint32 vertical_resolution_pix = 3; // Vertical resolution (in pixels)
     uint32 bit_rate_b_s = 4; // Bit rate (in bits per second)
     uint32 rotation_deg = 5; // Video image rotation (clockwise, 0-359 degrees)
+    float horizontal_fov_deg = 7; // Horizontal fov in degrees
```
Comment (julianoes) [inline protos/camera/camera.proto:299]: @JonasVautherin @markusachtelik is 7 above 6 on purpose here? Something to do with trimming?

Comment (markusachtelik) [reply protos/camera/camera.proto:299]: leftover from some experimenting, will change

File: protos/camera/camera.proto (hunk 3 above)
Comment (markusachtelik) [reply protos/camera/camera.proto]: `horizontal_fov_deg` changed to float. keeping `rotation_deg` to avoid breaking existing code

**Review by JonasVautherin — APPROVED** [review]

**Review by julianoes — APPROVED** [review]

**Pull request merged**